metrics/web/docs/features/remote_config/diagrams/remote_config_presentation_class.puml

```diff
@@ -0,0 +1,27 @@
+@startuml
+
+package base.domain.usecases {
+    class GetRemoteConfigUseCase{}
+}
+
+package common.presentation.state {
+    class RemoteConfigNotifier {
+        + isLoginFormEnabled : bool
+        + isFpsMonitorEnabled : bool
+        + isRendererDisplayEnabled : bool
+        + isLoading : bool
+
+        - _remoteConfig : RemoteConfig
+        - _getRemoteConfigUseCase : GetRemoteConfigUseCase
+        - Future<void> _initializeRemoteConfig()
+    }
+}
+
+package widgets {}
+
+RemoteConfigNotifier --> widgets : notifies
+widgets --> RemoteConfigNotifier : uses
```

Comment:
Not sure classes can interact (have relation) with a package on the class diagram. It is much better to provide a concrete example (say `CoolWidget` or `CoolPage`). Also, should we notice here that the relation between `LoadingPage`and notifier is mandatory?